src/Raven.Server/Smuggler/Documents/DatabaseSmuggler.cs

```diff
@@ -39,8 +39,7 @@ public class DatabaseSmuggler
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         internal static bool IsPreV4Revision(BuildVersionType buildType, string id, Document document)
         {
-            if (buildType == BuildVersionType.V3 == false)
```

Comment:
I do not see a good reason for changing this. Can we revert?

Reply:
done